### PR TITLE
docs(monitoring): add initial monitoring page with info on logs

### PR DESCRIPTION
**Pull request by douglas-reid**
An initial stab at monitoring + logs documentation.

I think we'll have to adjust this to make it more dev-friendly, but needed a starter that I can come back to with fresh eyes.

### CODE REVIEW

File: docs/api/steamship.agents.tools.rst [hunk 1]
```diff
@@ -15,6 +15,7 @@ Subpackages
    steamship.agents.tools.search
    steamship.agents.tools.speech_generation
    steamship.agents.tools.text_generation
+   steamship.agents.tools.video_generation
```
Comment (EniasCailliau) [inline docs/api/steamship.agents.tools.rst:18]: Thanks for adding these!

File: docs/developing/monitoring.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+.. code-block:: python
+
+   import logging
+   logging.info(f'running agent with prompt: {prompt}',
+                extra={'request_id': request_id, 'user': user}
+   )
+
+When a deployed instance is running, all logs at the ``INFO`` level and above will be generated, processed by Steamship,
+and made available for retrieval with a few minutes.
```
Comment (EniasCailliau) [inline docs/developing/monitoring.rst:67]:
```suggestion
and made available for retrieval within a few minutes.
```

**Review by EniasCailliau — DISMISSED** [review]
Clear and to the point. LGTM 📑🚀

File: docs/developing/monitoring.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+
+Monitoring via Web
+~~~~~~~~~~~~~~~~~~
+
+Steamship's website provides a hub for monitoring your packages and plugins. The monitoring page for
+an instance provides application logs, a detailed breakdown of usage, and a view of related
+:ref:`Tasks`.
+
+To access the monitoring page for an instance, visit the instance page. For example, for a package instance,
```
Comment (eob) [inline docs/developing/monitoring.rst:17]: Longer term I wonder if we'd be benefited by having a page called `Instance Page` that explains what it is and how to get there, and then we can just use the term Instance page as a link to that page

**Review by eob — DISMISSED** [review]
LGTM!

**Review by dkolas — DISMISSED** [review]
Nice, thank you for doing this!

**Pull request merged**